fighthealthinsurance/common_view_logic.py

```
@@ -559,24 +563,35 @@ def choose_appeal(
     ]:
         hashed_email = Denial.get_hashed_email(email)
         # Get the current info
-        denial = Denial.objects.filter(
+        denial: Denial = Denial.objects.filter(
```

Ruff (0.14.6)

`Denial` may be undefined, or defined from star imports

(F405)